### DISCUSSION TITLE
Importing IFC Files into iModelHub

### DISCUSSION THREAD

**Discussion by tangytang** [iTwin/itwinjs-core#3680] [Q&A]
Can I check what would be the most appropriate workflow for working with IFC files in iModelHub?

I understand that iTwin Synchroniser should work with commonly-used Design software such as Revit, AutoCAD, OpenBuilding etc(https://medium.com/itwinjs/revit-imodel-8f3ac68f1728). This allows the design models to be synched to iModelHub.

What if the IFC files are already generated? Can these IFC files be directly imported into iModelHub, or should it be 'processed' by a Design Software, and then synced to imodelHub?

Please advise.

**Comment by MichaelBelousov** [comment]
You can directly connect IFC 2x3 and IFC 4 files to your itwin. See [the synchronization documentation](https://developer.bentley.com/apis/synchronizations/supported-formats/) for the full list of supported formats.

Also you can see and manage which connectors you have at the iModel connectors settings in iTwin Synchronizer, but I believe they are added automatically if you try to add a new file format to a connection.
![image](https://user-images.githubusercontent.com/16315257/170259932-4f2b2b85-0f7d-4928-aab6-3c8b42c63c60.png)

which can be accessed by the settings gear in the top right:
![image](https://user-images.githubusercontent.com/16315257/170260002-4ea770b3-f386-439d-953c-4ee39c40c6bb.png)
